app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FoodsController < ApplicationController 
+  def index 
+    if params["q"].present?
+      @foods = UsdaFacade.foods_by_keyword(params["q"])
+      @total_hits = UsdaService.foods_by_keyword(params["q"])[:totalHits]
```

Comment:
Instead of calling the same method twice (and thus hitting the endpoint twice), line 5 could become
`@total_hits = @foods[:totalHits]`